units/host_termination.go

```diff
@@ -344,6 +344,13 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 			"job_type": j.Type().Name,
 		}))
 		return
+	} else {
```

Comment:
the else clause [is superfluous](https://golang.org/doc/effective_go.html#if): we will have returned on 346 if err != nil